share/completions/scp.fish

```diff
@@ -45,7 +45,7 @@ complete -c scp -d "Local Path" -n "not string match @ -- (commandline -ct)"
 
 # Remote path
 # Get the list of remote files from the scp target.
-string match -rq 'OpenSSH_(?<major>\d+)\.*' -- (ssh -V 2>&1)
+string match -rq 'OpenSSH_\w+(?<major>\d+)\.*' -- (ssh -V 2>&1)
```

Comment:
This should probably be `\w*`?

Reply:
Indeed, typo.

Reply:
I also changed for a more explicit regexp as you have recommend.